tox.ini

```diff
@@ -12,7 +12,7 @@ setenv =
 
 deps = -r{toxinidir}/test-requirements.txt
        -r{toxinidir}/requirements.txt
-commands = py.test --timeout=30 --cov-report html --cov-report=term --cov {toxinidir}/pipeline
+commands = py.test {posargs} --timeout=30 --cov-report html --cov-report=term --cov {toxinidir}/pipeline
```

Comment:
{posargs} lets you run individual tests from the command line with tox. For example:
tox test/test_pipeline_baseline.py::test_python_gapic_client_baseline
will run only that one test. Without it, the extra argument isn't parsed and all the tests run